bellboy/actors/realtime.py

```diff
@@ -0,0 +1,141 @@
+        self._websocket = websocket.create_connection(
+            f"wss://{url}",
+            sslopt={"cert_reqs": ssl.CERT_NONE},
+            options={"enable_multithread": False},
+        )
+        sleep(1)
+        self.log.info("Authenticating with the WebSocket service...")
+
+        # Call out until registered.
+        response = ""
+        while not response.startswith("REGISTERED"):
+            response = ""
+            self._websocket.send("BELLBOY")
+            self.log.info("Getting data from Realtime Services...")
+            response = self._websocket.recv()
+            self.log.info("Got data from Realtime Services: %s", response)
+            sleep(1)
+
+        # Announce successful setup:
+        self.log.info("Realtime Services are ready to go!")
+
+        # Get the ID saved by comms actor if the file is present
+        if self._identifier is None:
+            # Unpickle the auth file if it is present:
+            if (
+                os.path.exists(credential_path)
+                and os.path.isfile(credential_path)
+                and os.path.getsize(credential_path) > 0
+            ):
+                self.log.info("Found credential file, unpickling...")
+                auth_file = open(credential_path, "rb")
+                self._identifier = str(pickle.load(auth_file))
+                auth_file.close()
+
+    # --------------------------#
+    # MESSAGE HANDLING METHODS  #
+    # --------------------------#
+
+    def receiveMsg_str(self, message, sender):
+        """Sends a string as a message to NodeJS Services."""
+        if self._websocket is None:
+            self.log.error("Please authenticate before attempting to use this actor.")
+        else:
+            self.log.debug("Sending string %s to realtime logging service.", message)
+            if self._identifier:
+                self._websocket.send(f"BB{self._identifier}: {message}")
+            else:
+                self._websocket.send(message)
+
+    def receiveMsg_RealtimeCommsReq(self, message, sender):
+        """responding to simple sensor requests."""
+
+        self.log.info(
+            str.format("Received message {} from {}", message, self.nameOf(sender))
+        )
+
+        # ignore unauthorized requests
+        if sender != self.parent:
+            self.log.warning(
+                str.format("Received {} req from unauthorized sender!", message)
+            )
+            self.send(sender, Response.UNAUTHORIZED)
+
+        elif message == CommsReq.AUTHENTICATE:
+            self.authenticate()
+
+    def receiveMsg_Request(self, message, sender):
+        if message == Request.START:
+            self.log.info("STARTing WebSocket...")
+            self._setup_websocket()
+
+    def receiveMsg_SummaryReq(self, message, sender):
+        """sends a summary of the actor."""
+        self.send(sender, CommsResp.SUCCESS)
+
+    def summary(self):
+        self.log.info("Summary of realtime actor requested.")
```

Comment:
doesnt need to be an info log tbh, debug if anything

and same comment as before, u can remobe the method on 128, only this method on 132 needs to be filled out

Reply:
Removed.